reactive-messaging-http-quickstart/src/main/java/org/acme/reactivehttp/CostConverter.java

```diff
@@ -0,0 +1,34 @@
+package org.acme.reactivehttp;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A bean consuming costs in multiple currencies and producing prices in EUR from them
+ */
+@ApplicationScoped
+public class CostConverter {
+
+    private static final Map<String, Double> conversionRatios = new HashMap<>();
+
+    static {
+        conversionRatios.put("CHF", 0.93);
+        conversionRatios.put("USD", 0.84);
+        conversionRatios.put("PLN", 0.22);
+        conversionRatios.put("EUR", 1.);
```

Comment:
It might be just me, but I'm not a big fan of the `1.` notation. I found 1.0 clearly, but again, that might be me, so feel free to ignore it.

Reply:
I like `1.` more but I changed it for you :D

Reply:
:-D